security/db_router.py

```diff
@@ -0,0 +1,32 @@
+import sys
+
+from django.db.models.signals import pre_migrate, post_migrate
+
+from . import utils
+from .config import settings
+
+
+class MirrorSecurityLoggerRouter:
+    """
+    A router to control all that read and write database operation will be performed via mirror without atomic.
+    """
+
+    route_app_labels = [
+        'security', 'debug_toolbar_log'
+    ]
+
+    def db_for_read(self, model, **hints):
+        if not utils.is_running_migration and model._meta.app_label in self.route_app_labels:
```

Comment:
for migration is used default connection. In the future we will improve it to support more databases. But we will have to remove all migrations because of FK fields :(